### DISCUSSION TITLE
$ownerRecord not using global scopes

### DISCUSSION THREAD

**Discussion by danharper83** [filamentphp/filament#9952] [Help] [answered]
I seem to have an issue where global scopes are not applied to the $ownerRecord in the relationship manager. I have a global scope to exclude some large DB columns.

When using the relationship manager the scope is ignored, I can see in the debug bar a query for `select * from myModel where id = 1`  which means all the columns are available to the ownerRecord not just the scoped ones.

How would I exclude columns from the $ownerRecord?

**Comment by leandrocfe** [comment, marked as answer]
Maybe you can override the getOwnerMethod in the XXRelationManager.php file?
```php
public function getOwnerRecord(): Model
{
    return $this->getRecord(); //exclude columns...
}
```
